apps/etterlatte-saksbehandling-ui/client/src/components/behandling/beregningsgrunnlag/Beregningsgrunnlag.tsx

```diff
@@ -53,10 +56,15 @@ const Beregningsgrunnlag = () => {
       (barn) => barn.foedselsnummer !== behandling.søker?.foedselsnummer
     ) ?? []
 
-  const onSubmit = (beregningsgrunnlag: SoeskenMedIBeregning[]) =>
+  const onSubmit = (beregningsgrunnlag: SoeskenMedIBeregning[]) => {
```

Comment:
Nå vil onSubmit kalles hver gang man trykker `neste` fra Beregningsgrunnlag-siden. Må lage logikk for å kun poste endringer dersom det _er_ noen endringer

Reply:
La til et note på dette på jira-tasken, men ikke en stopper for videre utvikling. Kun ytelse og bruker-vennlighet-forbedringer